instrumentation/src/androidTest/java/com/bumptech/glide/DarkModeTest.java

```diff
@@ -8,6 +8,7 @@
 import android.content.Context;
 import android.content.res.Resources;
 import android.graphics.Bitmap;
+import android.graphics.Canvas;
```

Comment:
**Code Structure Issue**: The import 'android.graphics.Canvas' is added but not used in any of the modified code. It's good practice to remove unused imports to keep the codebase clean. <br> **Fix**: Remove the unused import to clean up the code. <br> **Code Suggestion**: 
 ```
 -import android.graphics.Canvas;
 ```